_sass/_base.scss

```
@@ -20,7 +20,7 @@
   }
 }
 
-a:not(.navbar-item) {
+a:not(.navbar-item,.no-underline) {
```

Add a space after the comma (if that's allowed in `not()` in CSS):

```suggestion
a:not(.navbar-item, .no-underline) {
```